pontoon/pretranslation/pretranslate.py

```diff
@@ -23,8 +23,8 @@ def get_translations(entity, locale):
         - plural form
         - user - tm_user/gt_user
     """
-    tm_user = User.objects.get(email="pontoon-tm@example.com")
-    gt_user = User.objects.get(email="pontoon-gt@example.com")
+    tm_user = User.objects.get(email="pontoon-tm@mozilla.com")
+    gt_user = User.objects.get(email="pontoon-gt@mozilla.com")
```

Comment:
It would be enough to revert the changes in both migration files. Changes in the remaining three files are OK.

Reply:
I had reverted all changes in my previous commit. In the latest commit i have added 2 new migration files and made changes in the remaining 3 files(pretranslate.py,
test_pretranslate.py,
test_tasks.py.)